files/es/web/api/batterymanager/chargingtime/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: BatteryManager.chargingTime
+slug: Web/API/BatteryManager/chargingTime
+tags:
+  - API
+  - Battery
+  - Battery API
+  - Mobile
+  - Referencia
+translation_of: Web/API/BatteryManager/chargingTime
+browser-compat: api.BatteryManager.chargingTime
+---
+{{deprecated_header}}
+
+{{APIRef("Battery API")}}
+
+Indica la cantidad de tiempo, en segundos, que faltan para que la batería esté completamente cargada.
+
+> **Nota:** Incluso si el tiempo devuelto es exacto al segundo, los navegadores los redondean a un intervalo más alto (típicamente a los 15 minutos más cercanos) por razones de privacidad.
+
+## Sintaxis
+
+```js
+var time = battery.chargingTime;
+```
+
+Cuando devuelve, `time` es el tiempo restante en segundos para que la batería
+esté completamente cargada o 0 (cero) si la `battery` ya está completamente cargada.
```

Comment:
`battery` está como código y veo que en la versión en inglés no

Reply:
sip, lo pusé asi por que deciá @sergiogval que bateria es un objecto por eso va como `battery` aún que no este así en inglés. o como ves?

Reply:
Acabo de ver que en la otra página en inglés si esta como código y aquí no XD creo que pueden quedar ambas como objeto, removiendo "la" antes de `battery`

Reply:
Creo que hay que dejar bien claras esas reglas y hacer un glosario de Objets, Methods and reserved keywords. Si no esto va a ser un problema de todos los días.